libs/zsdl/src/sdl2.zig

```diff
@@ -125,6 +125,10 @@ pub fn getVersion() Version {
 }
 extern fn SDL_GetVersion(version: *Version) void;
 
+test "compiled version should be same as linked version" {
+    try std.testing.expectEqual(VERSION, getVersion());
+}
```

Comment:
This is useful, but I think it might be too restrictive and/or confusing. If a user decides to discard the bundled SDL lib, the tests should still work.

I have been considering making the bundled SDL lib optional. SDL is fairly ubiquitous and ships backwards compatible improvements. IMO, the only real advantage to bundling it is for cross-compilation and this use case would be better served with stubs.

Reply:
I only added it so I could check tests actually work as there were none. We could just remove this or soften it to only check major

Reply:
Makes sense. How about moving it into it's own test step that can be run separately?

Reply:
Ya I can pull it out